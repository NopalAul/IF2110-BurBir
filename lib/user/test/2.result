lib/user/test/2.in lib/user/test/2.out: TRUE
