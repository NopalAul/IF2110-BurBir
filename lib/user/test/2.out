1
1
3
-1 1
gacor kang
deni
23/11/2023 21:52:75
1 2
gacor kang
leo
23/11/2023 21:52:75
2 3
gacor kang
yudi
23/11/2023 21:52:75
